File: Backend/src/main/java/com/xws/tim12/model/UserTokenState.java
package com.xws.tim12.model;

public class UserTokenState {

    private String jwtAccessToken;
    private Long expiresIn;

    public UserTokenState() {
        this.jwtAccessToken = null;
        this.expiresIn = null;
    }

    public UserTokenState(String jwtAccessToken, long expiresIn) {
        this.jwtAccessToken = jwtAccessToken;
        this.expiresIn = expiresIn;
    }

    public void setJwtAccessToken(String jwtAccessToken) {
        this.jwtAccessToken = jwtAccessToken;
    }

    public String getJwtAccessToken() {
        return jwtAccessToken;
    }

    public void setExpiresIn(Long expiresIn) {
        this.expiresIn = expiresIn;
    }

    public Long getExpiresIn() {
        return expiresIn;
    }

}

File: Frontend-Agent/src/app/models/userTokenState.ts
export class UserTokenState {

  public id: number;
  public role: string;
  public jwtAccessToken: string;
  public expiresIn: number;
  public username: string;

	constructor($id: number, $role: string, $jwtAccessToken: string, $expiresIn: number, $username: string) {
		this.id = $id;
		this.role = $role;
		this.jwtAccessToken = $jwtAccessToken;
		this.expiresIn = $expiresIn;
		this.username = $username;
	}

}